internal/configs/version1/nginx-plus.ingress.tmpl

```diff
@@ -107,6 +107,7 @@ server {
 		{{- end}}
 	{{- end}}
 	{{if $server.AppProtectDosName}}app_protect_dos_name "{{$server.AppProtectDosName}}";{{end}}
+    {{if $server.AppProtectDosAllowListPath}}app_protect_dos_access_file "{{$server.AppProtectDosAllowListPath}}";{{end}}
```

Comment:
Could you match the indentation with the surrounding block?

Reply:
fix in commit: 
[Code review fixes](https://github.com/nginxinc/kubernetes-ingress/pull/5824/commits/5c3eeeb2238cc007d0f9f8ee42e3d3ca89617f79)